runtime/src/bank_forks.rs

```diff
@@ -198,7 +198,16 @@ impl BankForks {
             last_accounts_hash_slot: root,
             in_vote_only_mode: Arc::new(AtomicBool::new(false)),
             highest_slot_at_startup: 0,
+        }));
+
+        for bank in bank_forks.read().unwrap().banks.values() {
```

Comment:
Not that it matters, but doesn't this set the fork graph of `LoadedPrograms` many times.

Reply:
It's not really typical usecase where `new_from_banks` is called. This is mostly for the safety of tests.